### PR TITLE
Adding USWDS replacement class names as needed.

### PR DESCRIPTION
**Pull request by ghost**
Closes department-of-veterans-affairs/vets.gov-team#830

Kinda sorta un-blocks department-of-veterans-affairs/vets.gov-team#549

### CODE REVIEW

**Comment by tiffanybbrown-va** [comment]
Goal here is to use both class names until the migration is complete. Then we can remove the Foundation class names.

**Comment by jcmeloni-usds** [comment]
LGTM - I did a spot check of probably half of the pages on the heroku deployment.  @melwoodard want to sanity check, too?

File: content/includes/4-action-page-questions.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="small-12 medium-4 columns">
+<div class="small-12 usa-width-one-third columns">
```
Comment (melwoodard) [inline content/includes/4-action-page-questions.html]: @tiffanybbrown-va this still needs to be combined the `medium-4` class for the time being, right?

File: src/js/disability-benefits/components/ClaimDetailLayout.jsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ export default class ClaimDetailLayout extends React.Component {
                 </div>
               </div>
             </div>
-            <div className="small-12 medium-4 columns">
+            <div className="small-12 usa-width-one-third columns">
```
Comment (melwoodard) [inline src/js/disability-benefits/components/ClaimDetailLayout.jsx]: @tiffanybbrown-va another spot that needs `medium-8` added back in

File: src/js/disability-benefits/containers/AdditionalEvidencePage.jsx [hunk 3]
```diff
@@ -116,7 +116,7 @@ class AdditionalEvidencePage extends React.Component {
                     onDirtyFields={this.props.setFieldsDirty}/>
               </div>
             </div>
-            <div className="small-12 medium-4 columns">
+            <div className="small-12 usa-width-one-third columns">
```
Comment (melwoodard) [inline src/js/disability-benefits/containers/AdditionalEvidencePage.jsx]: Needs `medium-8` added back in

File: src/js/disability-benefits/containers/AskVAPage.jsx [hunk 4]
```diff
@@ -84,7 +84,7 @@ class AskVAPage extends React.Component {
                 : null}
             </div>
           </div>
-          <div className="small-12 medium-4 columns">
+          <div className="small-12 usa-width-one-third columns">
```
Comment (melwoodard) [inline src/js/disability-benefits/containers/AskVAPage.jsx]: Needs `medium-4`

File: src/js/disability-benefits/containers/YourClaimsPage.jsx [hunk 5]
```diff
@@ -133,7 +133,7 @@ class YourClaimsPage extends React.Component {
                 cssClass="claims-upload-modal"
                 contents={<ConsolidatedClaims onClose={() => this.props.showConsolidatedMessage(false)}/>}/>
           </div>
-          <div className="small-12 medium-4 columns">
+          <div className="small-12 usa-width-one-third columns">
```
Comment (melwoodard) [inline src/js/disability-benefits/containers/YourClaimsPage.jsx]: Needs `medium-4`

File: src/js/facility-locator/containers/VAMap.jsx [hunk 6]
```diff
@@ -357,15 +357,15 @@ class VAMap extends Component {
           <SearchControls onChange={this.props.updateSearchQuery} currentQuery={currentQuery} onSearch={this.handleSearch}/>
         </div>
         <div className="row">
-          <div className="columns medium-4 small-12" style={{ maxHeight: '75vh', overflowY: 'auto' }} id="searchResultsContainer">
+          <div className="columns usa-width-one-third small-12" style={{ maxHeight: '75vh', overflowY: 'auto' }} id="searchResultsContainer">
```
Comment (melwoodard) [inline src/js/facility-locator/containers/VAMap.jsx]: Needs `medium-4`

**Review by melwoodard — CHANGES_REQUESTED** [review]
@tiffanybbrown-va looks pretty good, there were just a few spots where the medium breakpoint class was removed that need it added back.

**Comment by tiffanybbrown-va** [comment]
@melwoodard can you re-examine this when you have a chance?

File: src/js/disability-benefits/containers/ClaimEstimationPage.jsx [hunk 7]
```diff
@@ -53,7 +53,7 @@ class ClaimEstimationPage extends React.Component {
             <p>You can help speed up the process by promptly and electronically uploading the documents requested by VA.</p>
             <p>If you have questions, call VA at 855-574-7286, Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.</p>
           </div>
-          <div className="small-12 medium-4 columns">
+          <div className="small-12 usa-width-one-third medium-4columns">
```
Comment (melwoodard) [inline src/js/disability-benefits/containers/ClaimEstimationPage.jsx]: Needs an extra space here between `medium 4` and `columns`

File: src/js/disability-benefits/containers/DocumentRequestPage.jsx [hunk 8]
```diff
@@ -127,7 +127,7 @@ class DocumentRequestPage extends React.Component {
                     onDirtyFields={this.props.setFieldsDirty}/>
               </div>
             </div>
-            <div className="small-12 medium-4 columns">
+            <div className="small-12 usa-width-one-third medium-4columns">
```
Comment (melwoodard) [inline src/js/disability-benefits/containers/DocumentRequestPage.jsx]: Needs a space here too

File: src/js/gi/components/profile/HeadingSummary.jsx [hunk 9]
```diff
@@ -39,7 +39,7 @@ class HeadingSummary extends React.Component {
             <strong>{formatNumber(it.studentCount)}</strong> GI Bill students
             (<a onClick={this.props.onLearnMore}>Learn more</a>)
           </p>
-          <div className="small-12 medium-4 column">
+          <div className="small-12 usa-width-one-third medium-4column">
```
Comment (melwoodard) [inline src/js/gi/components/profile/HeadingSummary.jsx]: Space needed here too.

File: src/js/gi/components/profile/HeadingSummary.jsx [hunk 10]
```diff
@@ -56,7 +56,7 @@ class HeadingSummary extends React.Component {
               </IconWithInfo>
             </p>
           </div>
-          <div className="small-12 medium-4 column">
+          <div className="small-12 usa-width-one-third medium-4column">
```
Comment (melwoodard) [inline src/js/gi/components/profile/HeadingSummary.jsx]: Space needed here too.

File: src/js/gi/components/profile/HeadingSummary.jsx [hunk 11]
```diff
@@ -79,7 +79,7 @@ class HeadingSummary extends React.Component {
               </IconWithInfo>
             </p>
           </div>
-          <div className="small-12 medium-4 column"></div>
+          <div className="small-12 usa-width-one-third medium-4column"></div>
```
Comment (melwoodard) [inline src/js/gi/components/profile/HeadingSummary.jsx]: Space needed here too.

File: src/js/gi/containers/LandingPage.jsx [hunk 12]
```diff
@@ -60,7 +60,7 @@ export class LandingPage extends React.Component {
             </form>
           </div>
 
-          <div className="small-12 medium-4 columns">
+          <div className="small-12 usa-width-one-third medium-4columns">
```
Comment (melwoodard) [inline src/js/gi/containers/LandingPage.jsx]: Space needed here too.

**Review by melwoodard — CHANGES_REQUESTED** [review]
@tiffanybbrown-va just a few spaces missing between class names, otherwise looking good.

File: src/js/disability-benefits/containers/ClaimEstimationPage.jsx (hunk 7 above)
Comment (tiffanybbrown-va) [inline src/js/disability-benefits/containers/ClaimEstimationPage.jsx]: I'm annoyed with myself for not noticing that before committing.

**Review by melwoodard — APPROVED** [review]
This looks good now!

**Pull request merged**